src/modules/Dashboard/features/RespondentData/RespondentData.utils.ts

```diff
@@ -90,3 +90,18 @@ export const getConcatenatedEntities = <T extends Record<string, unknown>>({
 
   return concatenatedEntities;
 };
+
+export const getActivityWithLatestAnswer = <T extends { lastAnswerDate: string | null }>(
```

Comment:
Not needed anymore, because there is a function `getEntityWithLatestAnswer` which finds either an Activity or Flow with the latest answer.

Reply:
@IhorNesterenko99 thanks for the review and for catching these orphaned/duplicated functions / files.